### ISSUE TITLE
Design guideline/previews for WCAG Level AA recommendation

### ISSUE THREAD

**Issue by abacz** [cyfronet-fid/sat4envi#755]
Contrast 4,5:1 and 3:1. Check #750

**Comment by abacz** [comment]
Look & feel of WCAG high contrast 4,5:1

https://invis.io/S9RJNE8WXUB#/434415720_WCAG_-_Login
https://invis.io/S9RJNE8WXUB#/434415721_WCAG_-_Mapa-Produkt
https://invis.io/S9RJNE8WXUB#/434415722_WCAG_-_Modal
https://invis.io/S9RJNE8WXUB#/434415723_WCAG_-_Sentinel_Search
https://invis.io/S9RJNE8WXUB#/434415724_WCAG_-_Ustawienia

**Comment by AndrzejWojcik** [comment]
I've looked at the invisio prefs and for me, as a colorblind guy (green, brown, red, and blue issue) I have no problem with identifying any text, button, or content. So it should be OK.
